src/Raven.Server/Documents/Handlers/BatchRequestParser.cs

```diff
@@ -329,7 +330,7 @@ private static void ThrowMissingPatchProperty()
             BlittableJsonReaderObject reader;
             using (var builder = new BlittableJsonDocumentBuilder(ctx,
                 BlittableJsonDocumentBuilder.UsageMode.ToDisk,
-                key, parser, state))
+                key, parser, state, modifier: new StreamSource.BlittableMetadataModifier(ctx)))
```

Comment:
To where?
This way it is called only when we parse json from outside and not any where in our code which is not needed.